src/windows-emulator/emulator_thread.cpp

```diff
@@ -152,6 +155,10 @@ emulator_thread::emulator_thread(memory_manager& memory, const process_context&
             teb_obj.NtTib.Self = this->teb64->value();
             teb_obj.CurrentLocale = 0x409;
             teb_obj.ProcessEnvironmentBlock = context.peb64.value();
+            teb_obj.SameTebFlags.InitialThread = initial_thread;
+            teb_obj.SameTebFlags.SkipThreadAttach = static_cast<bool>(create_flags & THREAD_CREATE_FLAGS_SKIP_THREAD_ATTACH);
```

Comment:
you don't need the static cast :D

```suggestion
            teb_obj.SameTebFlags.SkipThreadAttach = create_flags & THREAD_CREATE_FLAGS_SKIP_THREAD_ATTACH;
```